src/main.js

```diff
@@ -75,6 +75,8 @@ function viewHomePage() {
   homeButton.classList.add('hidden');
   randomCoverButton.classList.remove('hidden');
   saveCoverButton.classList.remove('hidden');
+  formViewPage.classList.add('hidden');
```

Comment:
Good catch, Jenna, that the formViewPage was still showing when we clicked back to the homePage.